src/zenml/integrations/gcp/__init__.py

```diff
@@ -72,13 +77,17 @@ def flavors(cls) -> List[Type[Flavor]]:
             GCPImageBuilderFlavor,
             VertexOrchestratorFlavor,
             VertexStepOperatorFlavor,
+            VertexModelDeployerFlavor,
+            VertexAIModelRegistryFlavor,
```

Comment:
I'm not sure whether it's necessary, but the `VLLM` integration also imports the `services` in the `activate()` method

Reply:
it's not needed